server/services/project_admin_service.py

```diff
@@ -118,18 +152,18 @@ def update_project(project_dto: ProjectDTO, authenticated_user_id: int):
         if project_dto.license_id:
             ProjectAdminService._validate_imagery_licence(project_dto.license_id)
 
+        is_allowed_user = None
         if project_dto.private:
             ProjectAdminService._validate_allowed_users(project_dto)
+            is_allowed_user = username in project_dto.allowed_usernames
```

Comment:
Currently the allowed users can only map and validate, right? So on that case, I think we shouldn't give them permissions to edit the project.

Reply:
On that same file, let's update the `create_draft_project` function and allow also the organization members to create projects.

Reply:
> Currently the allowed users can only map and validate, right? So on that case, I think we shouldn't give them permissions to edit the project.

Right - looking at the current implementation, I can say this allowedUser list is only for mapping/validation. While editing(update_project), there is a check to validate:
* if there is atleast one user in the allowedUser list
* and if the username is valid (present in the DB)

Reply:
> On that same file, let's update the `create_draft_project` function and allow also the organization members to create projects.

Per discussion letting org managers create projects. Found a few issues on organisation endpoint. Sending that fix in a separate PR.